src/index.ts

```
@@ -236,23 +240,23 @@
 };
 
 export function stopMonitoring(activityNames?: string[]): void {
-  return ReactNativeDeviceActivityModule.stopMonitoring(activityNames);
+  return ReactNativeDeviceActivityModule?.stopMonitoring(activityNames);
```

Biome (1.9.4)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)